simpleui-server/src/props-file-reader.ts

```diff
@@ -221,10 +249,17 @@ export class PropsFileReader {
 
                 props['macro'][macroIndex]['index'] = macroIndex.toString();
                 props['macro'][macroIndex]['id'] = `macro-${macroIndex}`;
-                props['macro'][macroIndex]['token'] = macroParts[1];
-                props['macro'][macroIndex]['source'] = macroParts[4];
-                props['macro'][macroIndex]['property'] = macroParts[2];
-                PropsFileReader.processMacroPart(props, macroIndex);
+                props['macro'][macroIndex]['token'] = token;
+                props['macro'][macroIndex]['source'] = source;
+                props['macro'][macroIndex]['property'] = property;
+
+                // override dbname with given value
+                if (PropsFileReader.cmdVars.DbName !== "" && token.includes("_MYSQL_DB")) {
```

Comment:
I'd be inclined to use `token.endsWith("_MYSQL_DB")` rathen than `includes` to further restrict possible future accidental matches, but both should work ok at this point in time.

Reply:
Good catch. I like `endsWith` much  better than `includes`. I've updated it in my last commit